var/spack/repos/builtin/packages/easi/package.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+import os
+import shutil
+
+from spack import *
+from spack.cmd.pkg import GitExe
+
+
+class Easi(CMakePackage):
+    """easi is a library for the Easy Initialization of models
+    in three (or less or more) dimensional domains.
+    """
+
+    homepage = "https://easyinit.readthedocs.io"
+    git = "https://github.com/SeisSol/easi.git"
+
+    maintainers = ['ThrudPrimrose', 'ravil-mobile']
+
+    version('develop', branch='master')
+    version('1.0.1', tag='v1.0.1')
+
+    variant('asagi', default=True, description='build with ASAGI support')
+    variant('jit', default="impalajit", description='build with JIT support',
+            values=("impalajit", "impalajit-llvm"),
+            multi=False)
```

Comment:
I think you can add this:
```suggestion
    variant('jit', default="impalajit", description='build with JIT support',
            values=("impalajit", "impalajit-llvm"), when="target=x86_64:",
            multi=False)
```
and make the code in methods simpler.

Reply:
Sorry, re-reading the `package.py` - I think what keeps the current semantic is:
```
conflicts('jit=impalajit', when=`target=x86_64:`)
```
since `jit=impalajit-llvm` is allowed on other architectures.

Reply:
Hi,
I've just added `lua` as a variant together with a new version of `easi`. Now the variant looks like:
```
    variant('jit', default='impalajit', description='build with JIT support',
            values=('impalajit', 'impalajit-llvm', 'lua'),
            multi=False)
```

The legacy `impalajit` works only on `x86_64`. Both ` impalajit-llvm` and `lua` can work with other CPU architectures. I think `when="target=x86_64:"` will constraint everything to `x86` which we don't want i.e., we only need to restrict `impalajit`.

Reply:
You're right: see https://github.com/spack/spack/pull/28828#discussion_r805758714

Reply:
>conflicts('jit=impalajit', when=`target=x86_64:`)

Should it be this one?
```
conflicts('jit=impalajit', when=`target=aarch64:`)
conflicts('jit=impalajit', when=`target=ppc64:`)
conflicts('jit=impalajit', when=`target=ppc64le:`)
conflicts('jit=impalajit', when=`target=riscv64:`)
```
What do you think?

Reply:
Definitely.

Reply:
many thanks!. done.